include/contraction/contractionGraph.hpp

```diff
@@ -255,6 +354,37 @@ class contractionGraph :
         }
     }
 
+    /*! 
+        @brief builds the shortcut information and adds it during contraction or afterwards to copy them to the source graph
+    */
+    CH_edge process_shortcut_from_shortest_path(
```

Comment:
The name is quite confusing, we do not exploit any "shortest path" at this stage...

`process_shortcut`? `build_shortcut`? `build_shortcut_and_add_to_graph`?